cmd/blobovnicza-to-peapod/main.go

```diff
@@ -253,7 +253,7 @@ func migrateConfigToPeapod(dstPath, srcPath string) error {
 		return fmt.Errorf("encode modified config into YAML: %w", err)
 	}
 
-	err = os.WriteFile(dstPath, data, 0640)
+	err = os.WriteFile(dstPath, data, 0o640)
```

Comment:
whats the motivation to change prefix? anyway, separate commit for changed permissions pls